apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateLoadJobOptions.cs

```diff
@@ -216,6 +216,19 @@ internal void ModifyRequest(JobConfigurationLoad load)
             {
                 load.Encoding = Encoding;
             }
+            if (SessionID != null)
+            {
+                var cp = new ConnectionProperty
+                {
+                    Key = "session_id",
+                    Value = SessionID
+                };
+                load.ConnectionProperties = new ConnectionProperty[] { cp };
```

Comment:
Perhaps just change this to:

```csharp
load.ConnectionProperties = new[] { new ConnectionProperty { Key = "session_id", Value = SessionId };
```

Reply:
(And we might want to extract the constant out as a private constant in this class, with a comment referencing https://cloud.google.com/bigquery/docs/reference/rest/v2/ConnectionProperty.)